modules/ROOT/partials/configuration/toolbar_sticky.adoc

```diff
@@ -21,6 +21,6 @@ include::partial$misc/admon-different-default-for-mobile.adoc[]
 ----
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
-  toolbar_sticky: true
+  toolbar_sticky: true //toolbar_sticky: false will have no effect in inline mode.
```

Comment:
Not sure what the ticket dictates but I believe this would be better as a note below the heading with the name of the option. Something like `NOTE: In TinyMCE inline mode, this option cannot be set to False as the toolbar will always be sticky.`. And then include that the default value is `false` in "Classic" mode but `true` in Inline mode.

Reply:
This honestly should be a common admonition similar to features that only work inline mode. I mentioned that to @brian-forte but did so in a way that wasn't clear via https://github.com/tinymce/tinymce-docs/pull/2540

Reply:
I spent some time last night and this morning thinking on the page about this.

The thinking went something like this:

## suggested partial 1

### filename and path

`/tinymce-docs/modules/ROOT/partial/misc/admon-option-supported-in-iframe-mode-only.adoc`

### file contents

NOTE: `{productname}` has three main modes: _iframe_ mode (aka _classic_ or full editor mode); _inline_ editing mode; and _distraction-free_ editing mode (a specialised case of _inline_ mode). This option is only supported when `{productname}` is run in _iframe_ mode. It is not supported when `{productname}` is running in _inline_ or _distraction free_ mode. For more information on the differences between the editing modes, see xref:use-tinymce-inline.adoc[Inline editing mode].

## suggested partial 2

### filename and path

`/tinymce-docs/modules/ROOT/partial/misc/admon-option-supported-in-inline-mode-only.adoc`

### file contents

```asciidoc
NOTE: `{productname}` has three main modes: _iframe_ mode (aka
_classic_ or full editor mode); _inline_ editing mode; and
_distraction-free_ editing mode (a specialised case of _inline_ mode).
This option is only supported when `{productname}` is run in _inline_
mode. It is not supported when `{productname}` is running in _iframe_
mode. For more information on the differences between the editing
modes, see xref:use-tinymce-inline.adoc[Inline editing mode].
```

## deployment

1. These two files would replace the following two extant `partials`:

   * `/tinymce-docs/modules/ROOT/partials/misc/admon-inline-only.adoc`

   * `/tinymce-docs/modules/ROOT/partials/misc/admon-inline-editing-mode-does-not-support-comments.adoc`

2. All extant `includes` to these two files would be replaced by `includes` to the appropriate alternative.

3. Edits would be made to add case-specific extra info as required.

   For example, `modules/ROOT/partials/configuration/toolbar_sticky.adoc` could be edited as follows:
   
   ```asciidoc
   ### Example: Using `+toolbar_sticky+`

   [source,js]
   ----
   tinymce.init({
     selector: 'textarea',  // change this value according to your HTML
     toolbar_sticky: true
   });
   ----
   
   include::partial$misc/admon-option-supported-in-iframe-mode-only.adoc.adoc[]
   ```

## re-thinking 1

Because this is where things stop being as usefully neat on the re-use front.

@MitchCTiny’s [comment](https://github.com/tinymce/tinymce-docs/pull/2546/files#r998997340) suggests that, with regards `toolbar_sticky`, it’s less that this setting does or does not work with a given editing mode so much as the setting can only take one value in one mode, _inline_, and has different defaults depending on the mode.

Something like as follows:

```asciidoc
=== Example: Using `+toolbar_sticky+` in _iframe_ editing mode

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  toolbar_sticky: false  // this is the default value in iframe editing mode
});
----

When {productname} is running in _iframe_ editing mode (aka _classic_
or full editor mode), `+toolbar_sticky+` is set to `false` by default.
To dock the Toolbar and the menu to the top of the screen, change this
value to `true`.

=== Example: Using `+toolbar_sticky+` in _inline_ editing mode

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  toolbar_sticky: true   // this is the default value in inline editing mode; it cannot be changed
});
----

When {productname} is running in _inline_ editing mode,
`+toolbar_sticky+` is set to `true` by default and this value cannot
be changed. When {productname} is running in _inline_ editing mode,
Toolbars are only supported when in sticky or docking mode.
```

## re-thinking 2

Which reminded me of the other (smaller, yet real) issue I have with the sort of re-usable parapraphs I put together above: such paragraphs err on the side of being generic for the sake of being applicable in more circumstances.

The phrase **This option** used in the suggested partials above is a reference. It refers to whatever option is being discussed on the particular page said partial is included in. But the narrative distance between the reference and the referent is always something to pay attention to.

For example, my reference in the above paragraph — **the suggested partials** — is an enormous distance from the referent: 404 words and 23 paragraphs.

Way too far to be anything other than a good example of how bad using a reference in this way can be.

But even small narrative distances can lead to reader confusion.

For example, the existing partial, `admon-inline-editing-mode-does-not-support-comments.adoc`, is not especially re-usable, because it explicitly calls out Comments and the Sidebar. (NB: my reference in the sentence just passed is literally the filename of the thing. I’m referring to the object by its name. But, since I don’t include anything from the file, nor do I offer even directions to the file, this is a magnificently bad reference. It would take the reader literal minutes to track down the referenced object to and actually see what I’m talking about here.)

But, the text of `admon-inline-editing-mode-does-not-support-comments.adoc` is entirely clear about what is being discussed. If you only read that admonition, and ignore everything else on the page said admonition is `included` in, you still get the appropriate message.

The more generic **suggested partials** above rely on the reader connecting the NOTE with other material on the page. This is not, it turns out, a sensible thing to rely on. People pay as little attention to technical material as they can get away with. If they think reading one sentence is all they need to do, they will.

And pages with code-samples are particularly prone to this _read as little as I can_ approach. (Which is why my re-writes in the **re-thinking 1** section above include the comments they do.)

All of which is to argue for the approach taken in **re-thinking 1** above.

Reply:
Had a conversation with @kemister85 about a potential direction for this, and the process of creating and including partials.